src/ai/backend/manager/api/schema.graphql

```diff
@@ -564,6 +567,9 @@ type AgentNode implements Node {
   auto_terminate_abusing_kernel: Boolean
   local_config: JSONString
   container_count: Int
+
+  """Added in 25.1.0."""
```

Comment:
25.1.0 has already been released. Please update it here.

Reply:
Resolved